### ISSUE TITLE
tornado fails with 3.8.0b2 on Windows

### ISSUE THREAD

**Issue by lazka** [python/pyperformance#61]
```python
INFO:root:Running `C:\Python38\venv\cpython3.8-2459c2ec2c3d\Scripts\python.exe -u C:\Python38\venv\cpython3.8-2459c2ec2c3d\lib\site-packages\pyperformance\benchmarks\bm_tornado_http.py --output C:\Users\xy\AppData\Local\Temp\tmpdrgssiij`
Traceback (most recent call last):
  File "C:\Python38\venv\cpython3.8-2459c2ec2c3d\lib\site-packages\pyperformance\benchmarks\bm_tornado_http.py", line 96, in <module>
    runner.bench_time_func('tornado_http', bench_tornado)
  File "C:\Python38\venv\cpython3.8-2459c2ec2c3d\lib\site-packages\pyperf\_runner.py", line 458, in bench_time_func
    return self._main(task)
  File "C:\Python38\venv\cpython3.8-2459c2ec2c3d\lib\site-packages\pyperf\_runner.py", line 423, in _main
    bench = self._worker(task)
  File "C:\Python38\venv\cpython3.8-2459c2ec2c3d\lib\site-packages\pyperf\_runner.py", line 397, in _worker
    run = task.create_run()
  File "C:\Python38\venv\cpython3.8-2459c2ec2c3d\lib\site-packages\pyperf\_worker.py", line 293, in create_run
    self.compute()
  File "C:\Python38\venv\cpython3.8-2459c2ec2c3d\lib\site-packages\pyperf\_worker.py", line 331, in compute
    WorkerTask.compute(self)
  File "C:\Python38\venv\cpython3.8-2459c2ec2c3d\lib\site-packages\pyperf\_worker.py", line 280, in compute
    self.calibrate_loops()
  File "C:\Python38\venv\cpython3.8-2459c2ec2c3d\lib\site-packages\pyperf\_worker.py", line 243, in calibrate_loops
    self._compute_values(self.warmups, nvalue,
  File "C:\Python38\venv\cpython3.8-2459c2ec2c3d\lib\site-packages\pyperf\_worker.py", line 76, in _compute_values
    raw_value = self.task_func(self, self.loops)
  File "C:\Python38\venv\cpython3.8-2459c2ec2c3d\lib\site-packages\pyperf\_runner.py", line 454, in task_func
    return time_func(loops, *args)
  File "C:\Python38\venv\cpython3.8-2459c2ec2c3d\lib\site-packages\pyperformance\benchmarks\bm_tornado_http.py", line 60, in bench_tornado
    server, sock = make_http_server(make_application())
  File "C:\Python38\venv\cpython3.8-2459c2ec2c3d\lib\site-packages\pyperformance\benchmarks\bm_tornado_http.py", line 54, in make_http_server
    server.add_sockets(sockets)
  File "C:\Python38\venv\cpython3.8-2459c2ec2c3d\lib\site-packages\tornado\tcpserver.py", line 157, in add_sockets
    self._handlers[sock.fileno()] = add_accept_handler(
  File "C:\Python38\venv\cpython3.8-2459c2ec2c3d\lib\site-packages\tornado\netutil.py", line 268, in add_accept_handler
    io_loop.add_handler(sock, accept_handler, IOLoop.READ)
  File "C:\Python38\venv\cpython3.8-2459c2ec2c3d\lib\site-packages\tornado\platform\asyncio.py", line 79, in add_handler
    self.asyncio_loop.add_reader(
  File "C:\Python38\lib\asyncio\events.py", line 501, in add_reader
    raise NotImplementedError
NotImplementedError
```

**Comment by lazka** [comment]
See https://github.com/tornadoweb/tornado/pull/2686/files

**Comment by stonebig** [comment]
or this https://bugs.python.org/issue37373

**Comment by vstinner** [comment]
cc @methane

**Comment by vstinner** [comment]
I understand that a call to asyncio.set_event_loop_policy(asyncio.WindowsSelectorEventLoopPolicy()) should be added to bm_tornado.py on Windows with Python 3.8 or newer. Do you want to propose a pull request to implement that?

**Comment by lazka** [comment]
I've opened pull requests for all issues I've filed.

**Comment by vstinner** [comment]
> I've already opened pull requests for all issues I've filed.

Oh sorry. I looked at open issues before cooking 0.9.1 release, but I forgot to have a look at pull requests. I'm back from holiday, I didn't digest all my emails yet ;-)

**Comment by vstinner** [comment]
Should be fixed by merged PR #65.
